### ISSUE TITLE
Is there a better way to fine-tune yolov5s models after pruning

### ISSUE THREAD

**Issue by AFallDay** [VainF/Torch-Pruning#296]
I've tried putting the pruning code into train.py, but its detection is only half as good as the original, is there a better way? thanks

**Comment by VainF** [comment]
Hi @AFallDay, I'm not very familiar with Yolo. Using a smaller learning rate can be helpful.

**Comment by maolinLiu666** [comment]
> I've tried putting the pruning code into train.py, but its detection is only half as good as the original, is there a better way? thanks

Hello, I have also encountered this problem. Do you have a good solution? Please reply. Thank you
